components/account/AccountButton.tsx

```diff
@@ -77,34 +76,50 @@ const AccountButton: FC<{
             )}
           </div>
         ) : (
-          <div className="flex h-ztg-40">
+          <div
+            className={`flex flex-1	items-center justify-end h-full rounded-full cursor-pointer ${
+              pathname === "/" ? "bg-transparent border-white" : "border-black"
```

Comment:
Why different colours depending on route? Maybe we could add some opacity to get a similar effect and not have to worry about colours clashing with new banner. Something like twitter:
<img width="897" alt="image" src="https://user-images.githubusercontent.com/4950844/221163004-408c9ab5-df88-49fb-83f5-05d790edb973.png">

Reply:
I was going off the designs where the home page has different colours for the topbar components: https://www.figma.com/file/A0OEqjIWy6jL4Rk4kT8ddr/UX%2FUI_Workflow?node-id=2844%3A25907&t=5SVn1YlfoCddYx0T-0

But I do like the idea of adding opacity instead, it would also make it more consistent with the rest of the site. Are you suggesting something more like this?

![Screenshot 2023-02-26 at 10 36 20 PM](https://user-images.githubusercontent.com/14626415/221467205-f500a9e6-664b-401f-8329-d369fc944ff1.png)

Reply:
Yeah kind of though now I'm not sure! Maybe it should kick in further down the page? Though feel free to just leave this for now

Reply:
I added a transition animation to smooth it out but I'll circle back with Mikhail when I'm working on the mobile menu to get some input from him on this.